packages/x-date-pickers/src/TimeField/tests/editing.TimeField.test.tsx

```diff
@@ -341,6 +346,19 @@ describe('<TimeField /> - Editing', () => {
         keyStrokes: [{ value: '1', expected: '25' }],
       });
     });
+
+    it('should go to the next section when pressing `2` in a 12-hours format', () => {
```

Comment:
Could you add a test "I should set 3 when pressing 1 then 3" Otherwise the fix sounds good 👍

Reply:
Done :+1: